### PR TITLE
Added logging to kubernetes

### PR DESCRIPTION
**Pull request by BernardZhao**
Putting what I set up earlier into writing. Check out [github.com/ocf/loki](https://github.com/ocf/loki) to see the Loki setup that goes along with this.

### CODE REVIEW

**Comment by BernardZhao** [comment]
Just finished testing, it works perfectly after the tweaks. Thanks to @cg505 for letting me run puppet.

File: modules/ocf_kubernetes/files/logging.yaml [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    # <source>
+    #   @type tail
+    #   path /var/log/*.log
+    #   pos_file /var/log/fluentd/tmp/access.log.pos
+    #   tag foo.*
+
+    #   <parse>
+    #     @type json
+    #   </parse>
+    # </source>
```
Comment (cg505) [inline modules/ocf_kubernetes/files/logging.yaml:118]: can we clean this up

File: modules/ocf_kubernetes/files/logging.yaml [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+          pod
+        </label>
+        flush_interval 10s
+        flush_at_shutdown true
+        buffer_chunk_limit 1m
+      </store>
+
+      # <store>
+      #   @type stdout
+      # </store>
```
Comment (cg505) [inline modules/ocf_kubernetes/files/logging.yaml:180]: yeah remove this too imo

File: modules/ocf_kubernetes/files/logging.yaml (hunk 1 above)
Comment (BernardZhao) [reply modules/ocf_kubernetes/files/logging.yaml:118]: Ima keep it real with you chief idek what this is supposed to do

Comment (ja5087) [reply modules/ocf_kubernetes/files/logging.yaml:118]: It seems to be a comment? If it's not needed we can just remove it.

**Pull request merged**